index.js

```diff
@@ -0,0 +1,22 @@
+
+
+document.addEventListener("DOMContentLoaded", () =>{
+    const fecthAllCards = async () => {
+       let newDeck = await axios.get("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1")
```

Comment:
This all looks very good, but if you want to keep this data available even after running the function, you want to create these variables OUTSIDE this function. Otherwise you won't have this data available for future use.